loader/source/menu.c

```diff
@@ -1259,6 +1259,17 @@ static void Menu_Settings_InputHandler(MenuCtx *ctx)
 
 				// Set the value of the option in the config
 				ncfg->MeleeCodeOptions[item->identifier] = option->value;
+
+				// Update ncfg Video Width
+				if (item->identifier == MELEE_CODES_WIDESCREEN_OPTION_ID) {
+					if (!strcmp(option->name, "Off")) {

```

Comment:
probably want to disable the user's ability to edit the video width if widescreen is on. We might want to consider just disabling video width all together, I don't see a reason to use anything other than auto and 720.

Reply:
I can start on this and modify the description accordingly